examples/intRVFL.py

```diff
@@ -0,0 +1,539 @@
+        "Adult",
+        "Annealing",
+        "Arrhythmia",
+        "AudiologyStd",
+        "BalanceScale",
+        "Balloons",
+        "Bank",
+        "Blood",
+        "BreastCancer",
+        "BreastCancerWisc",
+        "BreastCancerWiscDiag",
+        "BreastCancerWiscProg",
+        "BreastTissue",
+        "Car",
+        "Cardiotocography10Clases",
+        "Cardiotocography3Clases",
+        "ChessKrvk",
+        "ChessKrvkp",
+        "CongressionalVoting",
+        "ConnBenchSonarMinesRocks",
+        "ConnBenchVowelDeterding",
+        "Connect4",
+        "Contrac",
+        "CreditApproval",
+        "CylinderBands",
+        "Dermatology",
+        "Echocardiogram",
+        "Ecoli",
+        "EnergyY1",
+        "EnergyY2",
+        "Fertility",
+        "Flags",
+        "Glass",
+        "HabermanSurvival",
+        "HayesRoth",
+        "HeartCleveland",
+        "HeartHungarian",
+        "HeartSwitzerland",
+        "HeartVa",
+        "Hepatitis",
+        "HillValley",
+        "HorseColic",
+        "IlpdIndianLiver",
+        "ImageSegmentation",
+        "Ionosphere",
+        "Iris",
+        "LedDisplay",
+        "Lenses",
+        "Letter",
+        "Libras",
+        "LowResSpect",
+        "LungCancer",
+        "Lymphography",
+        "Magic",
+        "Mammographic",
+        "Miniboone",
+        "MolecBiolPromoter",
+        "MolecBiolSplice",
+        "Monks1",
+        "Monks2",
+        "Monks3",
+        "Mushroom",
+        "Musk1",
+        "Musk2",
+        "Nursery",
+        "OocytesMerlucciusNucleus4d",
+        "OocytesMerlucciusStates2f",
+        "OocytesTrisopterusNucleus2f",
+        "OocytesTrisopterusStates5b",
+        "Optical",
+        "Ozone",
+        "PageBlocks",
+        "Parkinsons",
+        "Pendigits",
+        "Pima",
+        "PittsburgBridgesMaterial",
+        "PittsburgBridgesRelL",
+        "PittsburgBridgesSpan",
+        "PittsburgBridgesTOrD",
+        "PittsburgBridgesType",
+        "Planning",
+        "PlantMargin",
+        "PlantShape",
+        "PlantTexture",
+        "PostOperative",
+        "PrimaryTumor",
+        "Ringnorm",
+        "Seeds",
+        "Semeion",
+        "Soybean",
+        "Spambase",
+        "Spect",
+        "Spectf",
+        "StatlogAustralianCredit",
+        "StatlogGermanCredit",
+        "StatlogHeart",
+        "StatlogImage",
+        "StatlogLandsat",
+        "StatlogShuttle",
+        "StatlogVehicle",
+        "SteelPlates",
+        "SyntheticControl",
+        "Teaching",
+        "Thyroid",
+        "TicTacToe",
+        "Titanic",
+        "Trains",
+        "Twonorm",
+        "VertebralColumn2Clases",
+        "VertebralColumn3Clases",
+        "WallFollowing",
+        "Waveform",
+        "WaveformNoise",
+        "Wine",
+        "WineQualityRed",
+        "WineQualityWhite",
+        "Yeast",
+        "Zoo",
+    ]
+
+    def __init__(
+        self,
+        model_class: nn.Module,
+        simul: int = 1,
```

Comment:
Maybe a clearer variable name could be `repeats`?

Reply:
Agreed